README.md

```diff
@@ -1,238 +1,206 @@
-6. `spec.gslbLeader.controllerVersion`: The version of the GSLB leader cluster.
-7. `spec.gslbLeader.controllerIP`: The GSLB leader IP address or the hostname along with the port number, if any.
-8. `spec.memberClusters`: The kubernetes/openshift cluster contexts which are part of this GSLB cluster. See [here](#Multi-cluster kubeconfig) to create contexts for multiple kubernetes clusters.
-9.  `spec.refreshInterval`: This is an internal cache refresh time interval, on which syncs up with the AVI objects and checks if a sync is required.
-10. `spec.logLevel`: Specify the required types of logs that should be printed by AMKO. There are currently 4 supported types: `INFO`, `DEBUG`, `WARN` and `ERROR`.
-
-**Few Notes**:
-- Only one GSLBConfig object is allowed.
-- If using `helm install`, the GSLB Config object is created, just provide the right parameters in `values.yml`.
-- Once this object is defined and is accepted, it can't be changed (as of now). The only allowable edit is for the `logLevel` field. For all other fields, if changed, the changes will not take any effect. For the changes to take effect, one has to restart the AMKO pod.
-
-## Selecting kubernetes/openshift objects from different clusters
-A CRD called GlobalDeploymentPolicy allows users to select kubernetes/openshift objects based on certain rules. This GDP object has to be created on the same system wherever the GSLBConfig object was created and `amko` is running. The selection policy applies to all the clusters which are mentioned in the GDP object. A typical GlobalDeploymentPolicy looks like this:
-
-```yaml
-apiVersion: "amko.vmware.com/v1alpha1"
-kind: "GlobalDeploymentPolicy"
-metadata:
-  name: "global-gdp"
-  namespace: "avi-system"   // a cluster-wide GDP
-spec:
-  matchRules:
-    appSelector:
-      label:
-        app: gslb
-    namespaceSelector:
-      label:
-        app: gslb
- 
-  matchClusters:
-    - cluster: cluster1-admin    // cluster names are kubernetes cluster contexts
-    - cluster: cluster2-admin
- 
-  trafficSplit:
-    - cluster: cluster1
-      weight: 8
-    - cluster: cluster2
-      weight: 2
+kubectl delete gdp -n avi-system global-gdp
 ```
-1. `namespace`: an important piece here, as a GDP object created in `avi-system` namespace is recognised and all other GDP objects created in other namespaces are ignored.
-2. `matchRules`: List of selection policy rules. If a user wants to select certain objects in a namespace (mentioned in `namespace`), they have to add those rules here. A typical `matchRule` looks like:
-```yaml
-matchRules:
-    appSelector:                       // application selection criteria
-      label:
-        app: gslb                       // kubernetes/openshift label key-value
-    namespaceSelector:                 // namespace selection criteria
-      label:
-        ns: gslb                        // kubernetes/openshift label key-value
+Also, delete the `avi-system` namespace:
```

Comment:
Should we add a warning here that the AKO pod will also be deleted on this action?

Reply:
... if it is running in avi-system (since AKO supports dynamic namespaces).

Reply:
Changed this to:

Typically, an AKO instance could have been deployed in the `avi-system` namespace. But, if it is not, then the `avi-system` namespace can also be removed:
```
kubectl delete ns avi-system
```